src/tons_cli/job_group.py

```diff
@@ -17,8 +17,21 @@ def list_jobs():
     else:
         if len(jobs) == 0:
             click.echo(f"No jobs found")
-        for job in jobs:
-            click.echo(job)
+        for job_to_echo in jobs:
+            click.echo(job_to_echo)
+
+
+@job.command('list-active')
+def list_jobs():
+    """List all active jobs"""
+    jobs = sdk().get_all_active_jobs()
+    if jobs is None:
+        click.echo("Failed to get active jobs", err=True)
+    else:
+        if len(jobs) == 0:
+            click.echo(f"No active jobs found")
+        for job_to_echo in jobs:
+            click.echo(job_to_echo)
```

Comment:
```suggestion
        for job in jobs:
            click.echo(job)
```

Reply:
not sure you want to dump everything here to the user, filter out only what you see in the jobs screen (+latest APM)

Reply:
![image](https://user-images.githubusercontent.com/34208990/197381930-3e6ce858-d010-43ab-a0da-dec8238de75a.png)
 is this good?

Reply:
with APM:
![image](https://user-images.githubusercontent.com/34208990/197382188-b8f3cfcd-b403-4223-b328-22d5fdc508db.png)